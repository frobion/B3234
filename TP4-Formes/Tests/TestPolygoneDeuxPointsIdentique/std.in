PC PolyConvex1 1 1 3 -5 -2 -1
PC PolyConvex2 1 1 1 1 3 -5 -2 -1
LIST
EXIT

