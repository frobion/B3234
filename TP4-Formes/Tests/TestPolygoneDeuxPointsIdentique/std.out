OK
ERR
# Points alignes
PC PolyConvex1 1 1 3 -5 -2 -1
